### ISSUE TITLE
Feature Request: add an option to configure default directory to write lyrics to for lyrics plugin

### ISSUE THREAD

**Issue by doronbehar** [beetbox/beets#2806]
The title says it all. I think it could be great to have a `lyrics.default_directory` entry in the configuration file for a default directory in which the lyrics file will be written, instead of always specifying it manually with `-r` or `--write-rest`. Usually I run this command not from a specific directory so this could be useful.

**Comment by sampsyo** [comment]
Good point! Perhaps the simplest way to address this would be to just add a corresponding config option with the same semantics as the `-r` command-line flag.

**Comment by saniyafatima07** [comment]
@sampsyo @doronbehar Can I take up this issue under hacktober?

**Comment by doronbehar** [comment]
Yea sure. Good luck!

**Comment by saniyafatima07** [comment]
> Yea sure. Good luck!

@doronbehar 
Thank you for the reply 
Could you please label this issue with hacktober tag and assign it to me?

**Comment by doronbehar** [comment]
I don't have permission to label issues in this repo.

**Comment by saniyafatima07** [comment]
> I don't have permission to label issues in this repo.

No Problem, could you please tell me whom should I tag?

**Comment by saniyafatima07** [comment]
Sorry to bother again.
@sampsyo @snejus @doronbehar @bcotton 
Could this issue be labelled as hacktober and assigned to me or Could I know whom to tag ?

**Comment by hakkilab** [comment]
Hi, I'd like to work on this issue. I read through the plugin development docs and code a bit and it looks like whats desired is a config option like

```yaml
lyrics:
    rest_directory: <directory path>
```

Looks like this could be achieved by changing the option default from
```python
cmd.parser.add_option(
    "-r",
    "--write-rest",
    dest="rest_directory",
    action="store",
    default=None,
    metavar="dir",
    help="write lyrics to given directory as ReST files",
)
```
to 
```python
cmd.parser.add_option(
    "-r",
    "--write-rest",
    dest="rest_directory",
    action="store",
    default=self.config["rest_directory"].get(),
    metavar="dir",
    help="write lyrics to given directory as ReST files",
)
```
and adding a default like so
```python
self.config.add(
    {
        ...
        "print": False,
        "rest_directory": None,
        "synced": False,
        ...
    }
)
```

There would also need to be an update to the lyrics plugin docs describing the new config option.

Does this seem like it would work? This is my first attempt at contributing so any improvements or pointers are much appreciated.
